product/roundhouse.console/Program.cs

```diff
@@ -359,6 +359,10 @@ private static void parse_arguments_and_set_up_configuration(ConfigurationProper
                 .Add("isuptodate",
                      "This option prints whether there are any database updates or not, without actually running them. Other output except errors is disabled, to make it easy to use in scripts.",
                      option => { })
+                //do not store full text of run scripts
+                .Add("donotstorescriptsruntext",
+                     "DoNotStoreScriptsRunText - This instructs RH to not store the full script text in the database. Defaults to false.",
+                     option => configuration.DoNotStoreScriptsRunText = option != null)

```

Comment:
Is it better to name it "Positive"? i.e. `StoreScriptsRunText`, and default it to `true`? I think double negations are a bit hard to relate to (see also my comments further down, I think there is a negation too much in the code).

See e.g. https://softwareengineering.stackexchange.com/a/196837 for a discussion on affirmative (isXXX) vs negative (IsNotXXX) names on boolean variables.

Feel free to disagree, by all means :)

Reply:
I do agree on the affirmative naming. That's what I had initially (hence the forgotten negations), but it didn't quite feel consistent with `DoNotCreateDatabase` and `DoNotAlterDatabase`. That's why I changed it to the negative name.

I'll try to update my PR tomorrow :)

Reply:
Fair point. Maybe we should keep it as "DoNot". There are arguments for making the "unordinary" run cases a "true" flag too, if you see what I mean. So I may be convinced in both directions, really. So, keep the negative form if you like. But, please have a look at the concerns with the double negations of the flag in the code. I think the logic is reversed (but I might be wrong)

Reply:
You're right about the double negations. I've fixed them in the latest commit

Reply:
Thanks. Tested it locally, seems to work great. Thanks a lot for your contribution!!